portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path

```diff
@@ -355,8 +355,18 @@
 	<td></td>
 </tr>
 <tr>
-	<td>TOOLBAR_ANY_ITEM_BUTTON</td>
-	<td>//a[contains(@title,"${key_titleName}")]</td>
```

Comment:
Hello @Jamersonze you should not change existent PATHs due to the risk of breaking already existing tests which uses these paths. 
I've changed it back to this value and properly called it on your test.
Please see: https://github.com/emanuelfisilva/liferay-portal/pull/167/commits/80a8fc79bb17f6c461612fc84b590883d2e7b1ee